packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -277,6 +278,93 @@ function extractProxyAccessorsFor(
   ];
 }
 
+function getComputedKeyCompletion(path: NodePath<t.Expression>) {
+  path = skipTransparentExprWrappers(path);
+  if (path.isSequenceExpression()) {
+    const expressions = path.get("expressions");
+    return getComputedKeyCompletion(expressions[expressions.length - 1]);
+  }
+  return path;
+}
```

Comment:
Since this function is not meant for complex completion cases like `path.getCompletionRecord()` can handle, e.g. it does not support do expressions, I suggest rename it to `getComputedKeyLastElement` so that it is not confused with the completion concept in the spec.

Reply:
af8caa67c9c5f8e7a3e4a5dafdb453da29beea15